src/app/containers/RelatedTopics/index.jsx

```diff
@@ -0,0 +1,113 @@
+import React, { useContext } from 'react';
+import { TopicTag, TopicTags } from '@bbc/psammead-topic-tags';
+import { pathOr } from 'ramda';
+import SectionLabel from '@bbc/psammead-section-label';
+import styled from '@emotion/styled';
+import { arrayOf, shape, string, node } from 'prop-types';
+import { GelPageGrid, GridItemLarge } from '#components/Grid';
+import { ServiceContext } from '#app/contexts/ServiceContext';
+import { RequestContext } from '#app/contexts/RequestContext';
+import useClickTrackerHandler from '#hooks/useClickTrackerHandler';
+import useViewTracker from '#hooks/useViewTracker';
+import useToggle from '#hooks/useToggle';
+
+const eventTrackingData = {
+  componentName: 'topics',
+};
+
+const StyledTopicsWrapper = styled.aside`
+  padding-bottom: 2.5rem;
+`;
+
+const StyledSectionLabel = styled(SectionLabel)`
+  margin-top: 0;
+`;
+
+export const RelatedTopicsWrapper = ({ children }) => (
+  <GelPageGrid
+    enableGelGutters
+    columns={{
+      group0: 6,
+      group1: 6,
+      group2: 6,
+      group3: 6,
+      group4: 8,
+      group5: 20,
+    }}
+  >
+    <GridItemLarge>{children}</GridItemLarge>
+  </GelPageGrid>
+);
+
+RelatedTopicsWrapper.propTypes = {
+  children: node.isRequired,
+};
+
+const RelatedTopics = ({ topics }) => {
+  const { service, script, translations, dir } = useContext(ServiceContext);
+  const { isAmp, variant } = useContext(RequestContext);
+  const { enabled: topicTagsAreEnabled } = useToggle('topicsTags');
```

Comment:
Yeah I guess it doesn't add an awful amount of value, I can check with @greenc05 if testing locally is sufficient and if so I'll remove the toggle 👍

Reply:
Hello, it may be useful to be able to toggle this feature so UX can check the topics look ok across all services on test. I don't think it will be needed after go live.